php/example_code/lightsail/GetBlueprints.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * ABOUT THIS PHP SAMPLE => This sample is part of the SDK for PHP Developer Guide topic at
+ *
+ *
+ */
+// snippet-start:[lightsail.php.get_blueprints.complete]
+// snippet-start:[lightsail.php.get_blueprints.import]
+require 'vendor/autoload.php';
+
+use Aws\Exception\AwsException;
+use Aws\Lightsail\LightsailClient;
+
+// snippet-end:[lightsail.php.get_blueprints.import]
+
+/**
+ * Get a list of available EC2 instance images or blueprints from Amazon Lightsail.
+ *
+ * This code expects that you have AWS credentials set up per:
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html
+ */
+
+//Create a Lightsail Client
+// snippet-start:[lightsail.php.get_blueprints.main] 
+$client = new Aws\Lightsail\LightsailClient([
+    'profile' => 'default',
+    'version' => '2016-11-28',
+    'region' => 'us-east-2'
+]);
+
+try {
+    $result = $client->getBlueprints([
+        'includeInactive' => false,
+    ]);
+    foreach ($result['blueprints'] as $blueprint) {
+        print("<p>Blueprint - <b>" . $blueprint['blueprintId'] . "</b>: " . $blueprint['name'] . " version " . $blueprint['version'] . " for " . $blueprint['platform'] . "<br />");

```

Comment:
Is there a reason you're printing out HTML here (and in some of the other examples)? If there's not, I'd suggest removing the HTML tags from all of the examples to be more use-case-neutral.

Reply:
I was attempting to create a response that was a bit easier to read/and parse when using the example in the web browser. 

The Blueprint ID is needed when creating a new instance. This is an attempt to assist developers in getting the information they need to create a new instance. 

Thank you for your feedback.

Reply:
I'd probably aim for consistency here, and we don't include HTML tags in most of our examples, so shouldn't add them for specific cases unless there was a specific reason that this particular example needs it.